app/services/course_progress.rb

```diff
@@ -1,6 +1,6 @@
 # User's course progress and module state
 #
-class CourseProgress
+class ContentfulCourseProgress
   def initialize(user:)
     @user = user
```

Comment:
is this right?

Reply:
Nope. This was why I opted to make changes in the copied views so I could easily distinguish the duplicates.